src/theme/foundations/typography.ts

```diff
@@ -178,6 +183,9 @@ const typography: Typography = {
         subtitle1: fontSizeValues.mobile["subtitle1"],
         subtitle2: fontSizeValues.mobile["subtitle2"],
       },
+      tab: {
+        default: fontSizeValues.mobile["subtitle1"],
+      },
```

Comment:
Should this be `fontSizeValues.mobile["tab"]`? And same for the desktop value?

Reply:
Yes. And I actually tweaked it a bit in anticipation of the new variant for the `Tabs` component.